website/content/en/docs/getting-started/_index.md

```diff
@@ -73,6 +73,25 @@ provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication
 necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
 of communicating with identity providers. 
 
+### Tag Subdomains
+
+Karpenter expects subnets to be tagged with the cluster name for autodiscovery.
```

Comment:
Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.

```...```